artiq/coredevice/comm_kernel.py

```diff
@@ -635,13 +638,27 @@ def _serve_exception(self, embedding_map, symbolizer, demangler):
         python_exn.artiq_core_exception = core_exn
         raise python_exn
 
+    def _serve_async_error(self):
+        collision = self._read_bool()
+        busy_error = self._read_bool()
+        sequence_error = self._read_bool()
+        map_name = lambda y, z: [f"{y}(s)"] if z else []
+        errors = map_name("collision", collision) + \
+                 map_name("busy error", busy_error) + \
+                 map_name("sequence error", sequence_error)
+        logger.warning(f"During the lifetime of the kernel execution, it detected "
```

Comment:
"lifetime" is confusing (Rust lifetimes, NAC3 lifetimes) and too verbose. Just say "Errors X Z Y reported during kernel execution".